src/raw/aws/fargate_raw_metrics.go

```diff
@@ -0,0 +1,65 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/newrelic/infra-integrations-sdk/log"
+
+	"github.com/newrelic/nri-docker/src/raw"
+)
+
+func fargateRawMetrics(fargateStats FargateStats) map[string]*raw.Metrics {
+	rawMetrics := make(map[string]*raw.Metrics, len(fargateStats))
+
+	for containerID, stats := range fargateStats {
+		if stats == nil {
+			log.Debug("did not find container stats for %s, skipping", containerID)
+			continue
+		}
+		rawMetrics[containerID] = &raw.Metrics{
+			Time:        time.Time{},
+			ContainerID: containerID,
+			Memory: raw.Memory{
+				UsageLimit: stats.MemoryStats.Limit,
+				Cache:      stats.MemoryStats.Stats["cache"],
+				RSS:        stats.MemoryStats.Stats["rss"],
+				SwapUsage:  0,
```

Comment:
Fargate doesn't return swap usage. It can be confirmed through the JSONs that are also part of this PR. They have the raw responses from the Metadata v3 API.